infra/utils.py

```diff
@@ -71,7 +71,7 @@ def execute(command, location=None, check_result=False):
   return out, err, process.returncode
 
 
-def get_fuzz_targets(path):
+def get_fuzz_targets(path, root_only=False):
```

Comment:
s/root_only/top_level_only/ ? `root_only` confused me (root as in the root user?) for a bit while I was reviewing until I saw this function. 

also need to update docstring.

Reply:
Done.
What do you think of never recursively searching subdirs for fuzz targets?
I guess if coverage doesn't work with fuzz targets in subdirs (because there will be binaries that are false positives that get put in out in cov builds when source code is copied to out), we may was well not support it elsewhere right?

Reply:
What would it take to support fuzz targets in subdirs for coverage reports? 

Not supporting fuzz targets in subdirs is inconsistent with OSS-Fuzz and may cause trouble down the line.

Reply:
Right now coverage uses [this function](https://github.com/google/oss-fuzz/blob/master/infra/helper.py#L692) for determining which corpora to download and [this bash code](https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-runner/coverage#L22) to determine which fuzz targets to run. Both of those do not search subdirectories, so I think CIFuzz's behavior here is consistent with OSS-Fuzz's (even if coverage's behavior is inconsistent with ClusterFuzz's behavior).

I think I could try to merge all of these, but I think that's best left for another PR.

Reply:
Funny coincidence: https://github.com/google/oss-fuzz/pull/5959